### DISCUSSION TITLE
need help on github CI

### DISCUSSION THREAD

**Discussion by deep-soft** [rustdesk/rustdesk#5575] [General]
can't fix this error:
cp: cannot stat '.res': No such file or directory

https://github.com/rustdesk/rustdesk/pull/5574#issuecomment-1702877144

**Comment by rustdesk** [comment]
Frankly, I do not know either.

**Comment by deep-soft** [comment]
@rustdesk 
I found the error in flutter-build.yml
How can I force another pull request for the branch if already exist one?
